fMain.cs

```diff
@@ -88,20 +88,20 @@ private void SetOptions(string[] O)
             int Checks = (int)Base36.Decode(O[1]);
             int Combos = (int)Base36.Decode(O[2]);
             int Colour = (int)Base36.Decode(O[3]);
-            if ((Checks & 8192) > 0) { cUserItems.Checked = true; } else { cUserItems.Checked = false; };
-            if ((Checks & 4096) > 0) { cAdditional.Checked = true; } else { cAdditional.Checked = false; };
-            if ((Checks & 2048) > 0) { cGossip.Checked = true; } else { cGossip.Checked = false; };
-            if ((Checks & 1024) > 0) { cSoS.Checked = true; } else { cSoS.Checked = false; };
-            if ((Checks & 512) > 0) { cSpoiler.Checked = true; } else { cSpoiler.Checked = false; };
-            if ((Checks & 256) > 0) { cMixSongs.Checked = true; } else { cMixSongs.Checked = false; };
-            if ((Checks & 128) > 0) { cBottled.Checked = true; } else { cBottled.Checked = false; };
-            if ((Checks & 64) > 0) { cDChests.Checked = true; } else { cDChests.Checked = false; };
-            if ((Checks & 32) > 0) { cShop.Checked = true; } else { cShop.Checked = false; };
-            if ((Checks & 16) > 0) { cDEnt.Checked = true; } else { cDEnt.Checked = false; };
-            if ((Checks & 8) > 0) { cBGM.Checked = true; } else { cBGM.Checked = false; };
-            if ((Checks & 4) > 0) { cEnemy.Checked = true; } else { cEnemy.Checked = false; };
-            if ((Checks & 2) > 0) { cCutsc.Checked = true; } else { cCutsc.Checked = false; };
-            if ((Checks & 1) > 0) { cQText.Checked = true; } else { cQText.Checked = false; };
+            cUserItems.Checked = (Checks & 8192) > 0 ? true : false;
```

Comment:
Since you're simplifying it, can this just be `cUserItems.Checked = (Checks & 8192) > 0;`. same for below